lib/HalTtgoTDisplay/Board.h

```diff
@@ -64,6 +64,36 @@ namespace Board
 /** Pin number of all used pins. */
 namespace Pin
 {
+#if defined(BOARD_LILYGO_T_DISPLAY_S3)
+    /** Pin number of onboard LED */
+    constexpr uint8_t    onBoardLedPinNo         = IoPin::NC;  // Not available
+
+    /** Pin number of I2C SDA */
+    constexpr uint8_t    i2cSdaPinNo             = 2U;
+
+    /** Pin number of I2C SDL */
+    constexpr uint8_t    i2cSclPinNo             = 3U;

```

Comment:
According to schematic and examples the i2c scl should be 17.

Reply:
fixed as suggested.